primitives/src/v1/mod.rs

```diff
@@ -82,6 +83,12 @@ pub mod well_known_keys {
 	//     <Hrmp as Store>::HrmpEgressChannelsIndex::prefix_hash();
 	//
 
+	/// The current slot number.
+	///
+	/// The storage entry should be accessed as a `Slot` encoded value.
+	pub const CURRENT_SLOT: &[u8] =
+		&hex!["1cb6f36e027abb2091cfb5110ab5087f06155b3cd9a8c9e5e9a23fd5dc13a5ed"];
```

Comment:
Nit: This is a bit unfortunate that we don't have a test for this. 

The problem I see is that `sp_consensus_slot` (I assume that modules hosts the storage value) doesn't have any indication that the value is accessed externally and that seems fragile.

Maybe we could move the const to the pallet near to where the value lives, write the test there and make it a pub export, and here we just re-export it?

Reply:
Yeah, I'll do that. This key is coming from BABE so I'll move this `const` to substrate, add a test, and then re-export it here as you suggested.

Reply:
No, please don't move this to substrate! This is highly polkadot specific, especially because it depends on the naming of the pallet in the runtime.

We can add a test and improve this stuff more in the future.